src/hubs/lpf2hub.ts

```diff
@@ -410,7 +410,8 @@ export class LPF2Hub extends Hub {
         const device = this._getDeviceByPortId(portId);
 
         if (device) {
-            device.receive(message);
+            // Forward message to device (without size, hub and port)
+            device.receive(message.slice(3));
```

Comment:
Device does not need to get the full message, data is sufficient.
I did not test and I wonder if it should be 4 instead of 3.

Reply:
Also, it seems the offset is not the same for WEDO2, it would remove specifique parsing from devices.